src/pages/publication/contributors_tab/AddContributor.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button } from '@material-ui/core';
+
+import Modal from '../../../components/Modal';
+import { Authority } from '../../../types/authority.types';
+import AddContributorModal from './components/AddContributorModal';
+
+const AddContributor: FC = () => {
+  const { t } = useTranslation('publication');
+
+  const [addedAuthors, setAddedAuthors] = useState<Authority[]>([]);
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const addAuthor = (author: Authority) => {
+    setAddedAuthors([...addedAuthors, author]);
+    handleClose();
+  };
+
+  return (
+    <>
+      {addedAuthors && addedAuthors.map(author => <h4 key={author.systemControlNumber}>{author.name}</h4>)}
+      <Button onClick={handleOpen} variant="contained" color="primary" data-testid="add-contributor">
+        {t('contributors.add_author')}
+      </Button>
+      {open && (
+        <Modal
+          ariaDescribedBy=""
+          ariaLabelledBy=""
+          headingText={t('contributors.add_author')}
+          onClose={handleClose}
+          openModal={open}>
+          <AddContributorModal addAuthor={addAuthor} />
```

Comment:
Kutte `Modal`-postfix? Nå virker det som at det er to nesting av modaler

Reply:
ja, kan kalle det AddContributorContent? eller AddContributorModalContent?